### PR TITLE
[wasm] Small cleanup of main wasm directory

### PR DESCRIPTION
**Pull request by kjpou1**
- Move source support files for custom building the wasm runtimes to a new directory called `wasm/src`
- Modified `packager.exe` to build from this new subdir.
- Updated the README.md to document the change.

### CODE REVIEW

File: sdks/wasm/Makefile [hunk 1]
```diff
@@ -112,10 +112,10 @@ toolchain: .stamp-jsvu
 emsdk_env.sh:
 	cd $(EMSCRIPTEN_SDK_DIR) && ./emsdk construct_env $(CURDIR)/emsdk_env.sh
 
-driver-dynamic.o: driver.c corebindings.c pinvoke-tables-default.h emsdk_env.sh
-	$(EMCC) -g -Os -fPIC -s WASM=1 -s ALLOW_MEMORY_GROWTH=1 -s BINARYEN=1 -s "BINARYEN_TRAP_MODE='clamp'" -s ALIASING_FUNCTION_POINTERS=0 -DWASM_SUPPORTS_DLOPEN -DCORE_BINDINGS -I$(WASM_RUNTIME_DIR)/include/mono-2.0 driver.c -c -o driver-dynamic.o
+driver-dynamic.o: support/driver.c support/corebindings.c support/pinvoke-tables-default.h emsdk_env.sh
+	$(EMCC) -g -Os -fPIC -s WASM=1 -s ALLOW_MEMORY_GROWTH=1 -s BINARYEN=1 -s "BINARYEN_TRAP_MODE='clamp'" -s ALIASING_FUNCTION_POINTERS=0 -DWASM_SUPPORTS_DLOPEN -DCORE_BINDINGS -I$(WASM_RUNTIME_DIR)/include/mono-2.0 support/driver.c -c -o driver-dynamic.o
```
Comment (vargaz) [inline sdks/wasm/Makefile]: Would suggest naming the directory 'src'.

Comment (kjpou1) [reply sdks/wasm/Makefile]: 👍 Will do

Comment (kjpou1) [reply sdks/wasm/Makefile]: Addressed in recent commit

**Comment by vargaz** [comment]
We might also want to rename library_mono.js to something else, preferably with a - instead of a _.

**Pull request merged**